src/components/atoms/AntFormItemWrapper.tsx

```diff
@@ -8,7 +8,7 @@ import styles from "./antFormItemWrapper.module.css";
 interface AntFormItemWrapperProps {
   children: ReactElement;
   className?: string;
-  label?: string;
+  label?: string | JSX.Element;
```

Comment:
prefer the type `ReactNode` here (see [https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement](https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement)